frontend/dockerfile/dockerfile_mount_test.go

```diff
@@ -468,3 +470,54 @@ COPY --from=base /tmpfssize /
 	require.NoError(t, err)
 	require.Contains(t, string(dt), `size=131072k`)
 }
+
+func testMountDuplicate(t *testing.T, sb integration.Sandbox) {
```

Comment:
Leave a comment `// moby/buildkit#4123` here so it is clear that this is a regression test.